transforms/code/header_cleanser/README.md

```diff
@@ -0,0 +1,11 @@
+# Legal Removal Transform 
```

Comment:
Can you update the readme with new name

Reply:
Now needs to be Header Cleanser Transform